polygon-validator-signer-balance-bot/src/agent.spec.ts

```diff
@@ -0,0 +1,161 @@
+import { FindingType, FindingSeverity, Finding, getJsonRpcUrl, ethers, HandleBlock, Initialize } from "forta-agent";
+import { provideHandleBlock, provideInitialize } from "./agent";
+import { TestBlockEvent } from "forta-agent-tools/lib/test/index";
+import { createAddress } from "forta-agent-tools/lib/utils";
+import { Flag, MINIMUM_THRESHOLD_TO_ETHER } from "./constants";
+
+describe("POLYGON-VALIDATOR-SIGNER BOT TEST SUITE", () => {
+  let handleBlock: HandleBlock;
+  let mockEthersProvider: any;
+  let initialize: Initialize;
+  let flag: Flag;
+  let mockUser: string;
+
+  beforeEach(async () => {
+    mockEthersProvider = {
+      getBalance: jest.fn(),
+    };
+    flag = { wasOverThresholdAlert: false };
+
+    mockUser = createAddress("0x01");
+    initialize = provideInitialize(mockEthersProvider as any, flag, mockUser);
+    handleBlock = provideHandleBlock(mockEthersProvider as any, flag, mockUser);
```

Comment:
```suggestion
    initialize = provideInitialize(mockEthersProvider, flag, mockUser);
    handleBlock = provideHandleBlock(mockEthersProvider, flag, mockUser);
```

The provider is already of type `any`.